common/utils/validator-utils.js

```diff
@@ -1,24 +1,11 @@
 import { passwordStrengthRegex } from '../constants/validations';
 
-/**
- * Take a string and determine if it has non-whitespace characters in it
- *
- * This may seem counter-intuitive, but it's difficult to get a zipcode regex correctly.
- * See https://stackoverflow.com/questions/578406/what-is-the-ultimate-postal-code-and-zip-regex
- * for more info
- *
```

Comment:
I would leave `isValidZipcode` alone if for no other reason than because of these comments. If it's removed, I'm sure we'll have people popping up asking about our lack of validation. While this function doesn't do much (it simply prevents whitespace submissions), it exists and says to most: "we have some sort of validation on zipcodes" and it explains why it's weak for anybody that dives deeper.

Reply:
The validation can be accomplished via `Yup.string().trim()`. I can add it back though if you want!

Reply:
The validation can be accomplished via Yup.string().trim(). I can add it back though if you want!

Reply:
I guess that's fair then, please move the comments over too